microcosm_flask/tests/test_basic_auth.py

```diff
@@ -32,14 +38,12 @@ def unauthorized():
     response = client.get("/unauthorized")
     assert_that(response.status_code, is_(equal_to(401)))
     data = loads(response.get_data().decode("utf-8"))
-    assert_that(data, is_(equal_to({
-        "code": 401,
-        "message": "The server could not verify that you are authorized to access the URL requested. "
-                   "You either supplied the wrong credentials (e.g. a bad password), or your browser "
-                   "doesn't understand how to supply the credentials required.",
-        "retryable": False,
-        "context": {"errors": []},
-    })))
+    assert_that(data, has_entries(dict(
+        code=401,
+        message=starts_with("The server could not verify that you are authorized to access the URL requested"),
```

Comment:
The exact message has a different number of spaces locally vs. on CI. Testing for the whole message is not particularly interesting. Using `starts_with` is enough to make sure we report the error message.